src/type/__tests__/validation-test.ts

```diff
@@ -341,6 +341,12 @@ describe('Type System: A Schema must have Object root types', () => {
       input SomeInputObject {
         test: String
       }
+      input SomeInputObject2 {
+        test: String
+      }
+      input SomeInputObject3 {
+        test: String
+      }
```

Comment:
For context: I changed this test to avoid getting an error here for reusing root types as well, to keep the scope of the test as tight as possible.